src/main/java/svc/controllers/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,27 @@
+package svc.controllers;
+
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.dao.IncorrectResultSizeDataAccessException;
+import org.springframework.http.HttpStatus;
+
+@ControllerAdvice
+@RestController
+public class GlobalExceptionHandler {
+	
+	@ResponseStatus(HttpStatus.NOT_FOUND)
+	@ExceptionHandler(IncorrectResultSizeDataAccessException.class)
+	public String handleIncorrectResultSizeDataAccessException(IncorrectResultSizeDataAccessException e){
+		return ("Result Not Found");
+	}
+	
+	@ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
We should not return NOT_FOUND for generic "Exception"s. I would return a 500 Internal Server Error since it's unknown. It definitely needs to be 5xx